src/bp/ui-studio/src/web/views/OneFlow/diagram/VariableTypesForm/EnumForm.tsx

```diff
@@ -75,6 +78,28 @@ const EnumForm: FC<Props> = ({
     updateFormItem(convertFromTags(data))
   }
 
+  const isDupplicate = (localItem: Item[], newItem: Item) => {
+    const lastAdded: string = [newItem.name].concat(newItem.tags).slice(-1)[0]
```

Comment:
ES6. `[newItem.name, ...newItem.tags]`

Reply:
Changed for `[...[newItem.name], ...newItem.tags]`

Reply:
Why [...[newItem.name] ? No need for the spread when you only have one element